janus_core/single_point.py

```diff
@@ -267,6 +279,9 @@ def run_single_point(
         if write_kwargs and "filename" not in write_kwargs:
             raise ValueError("'filename' must be included in write_kwargs")
 
+        if self.logger is not None:
```

Comment:
Can just be:
```suggestion
        if self.logger:
```

Reply:
Could also configure an alias to this above when you define the logger.
**N.B** should go in `if` above one line here to save me formatting
```python
loginfo = self.logger.info if self.logger else lamba *x, **y: None
```
or some other nop

Reply:
I quite like the alias, but I think it might be better to stick with the more explicit form.